src/components/Dashboard/CrossingStatusHistory/InfiniteCrossingStatusHistoryList.js

```diff
@@ -114,35 +114,29 @@ export default class InfiniteCrossingStatusHistoryList extends React.Component {
     virtualizingList = allStatusUpdates.edges;
 
     return (
-      <div>
+      <div style={{ height: 'calc(100vh - 140px)' }}>
```

Comment:
@Niicck this is why we have the strange behavior on the crossing details page. This is just hardcoded to work with the size of header on the all crossing history page.